### ISSUE TITLE
validateIndentation: check multiline conditionals

### ISSUE THREAD

**Issue by ava-dylang** [jscs-dev/node-jscs#995]
Indent validation isn't validating multiline conditionals.

`.jscs.json`
```
{
    "validateIndentation": 4
}
```

### Valid Script
```
var condition_alpha = true,
    condition_beta = true
;

if (condition_alpha
    && condition_beta
) {
    console.log('Valid')
}
```
### Invalid Script
```
var condition_alpha = true,
    condition_beta = true
;

if (condition_alpha
   && condition_beta
) {
    console.log('Valid')
}
```
*Note: Line 6, char 3. Only three space indent, Ignored by intent validation.*

I think this is a bug in the intent validation, but It could also be called a feature request for a require/disallow multi-line conditional rulesets;

JSCS Version: `1.10.0`
Installed via nodejs on Ubuntu Server 12.04 LTS

**Comment by mikesherov** [comment]
Thanks for contributing! There is currently no plans for this rule, which is a feature request.

The reason this isn't validated in the validateIndentation rule is because there isn't a common accepted best practice for how the test should be indented.

**Comment by hzoo** [comment]
We released [3.0](https://github.com/jscs-dev/node-jscs/releases/tag/v3.0.0) recently! We also wrote a [post](https://medium.com/@markelog/jscs-end-of-the-line-bc9bf0b3fdb2#.oot3wvgpu) about how 3.0 will be our last major release (since we are deprecating the project) and that we are [merging with ESLint](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint)!

Thus we won't be working on any new features moving forward which includes new rules/options. We will continue to fix bugs and work on CST/better autofixing for current rules in the upcoming months both to help JSCS users and to gain experience that we can hopefully use in ESLint. But otherwise we both teams will be working on getting [ESLint feature parity with JSCS](https://github.com/eslint/eslint/milestones/JSCS%20Compatibility) and just a nice transition for our users.

I'm putting an `orphaned` label to note this issue isn't planned to be worked on, but might be a good contribution as a custom rule (not in JSCS core itself).

I would encourage anyone who wants the rule to try writing it themselves: I would either write a custom rule or submit an issue to ESLint and if accepted a PR. Otherwise a custom rule that won't be in core repo itself.

JSCS rules are also very similar to ESLint rules: here are some resources to get started: 

- [Project-specific Lint Rules with ESLint](https://gist.github.com/jareware/7179093)
- [Creating Custom Rules for ESLint](https://insideops.wordpress.com/2015/12/08/creating-custom-rules-for-eslint)
- [Maintaining JavaScript Code Quality with ESLint ](https://www.paypal-engineering.com/2014/12/12/maintaining-javascript-code-quality-with-eslint/)
